README.md

```diff
@@ -47,42 +47,49 @@ You can find a full list of our demos and view the code for each of them in the
 Full documentation is available at [https://geofirestore.com](https://geofirestore.com). It mostly provides the same functionality as the Firestore library, in almost the same way as the Firestore library. Many questions can be addressed by looking at the [Firestore docs](https://firebase.google.com/docs/firestore/). However there are a few differences, and below is a little example of how to make a location based query.
 
 ```TypeScript
```

Comment:
If you want I'd say use the JS the entire file. No need to make a distinction between JS/TS as the syntax is the same, just not the typings.